#!/usr/bin/env python
PACKAGE = "aerial_robot_base"

from dynamic_reconfigure.parameter_generator_catkin import *

KALMAN_FILTER_FLAG = 0
INPUT_SIGMA = 1
MEASURE_SIGMA = 2

gen = ParameterGenerator()

gen.add("kalman_filter_flag", bool_t, KALMAN_FILTER_FLAG, "Kalman Filter Flag", False)
gen.add("input_sigma", double_t, INPUT_SIGMA, "Baro Bias Sigma", 0.001, 0, 1)
gen.add("measure_sigma", double_t, MEASURE_SIGMA, "Baro Sigma", 0.05, 0, 1)

exit(gen.generate(PACKAGE, "kalman_filter", "KalmanFilterBaroBias"))


